### DISCUSSION TITLE
It would be nice to return a non-empty array when there is only one row in the sheet

### DISCUSSION THREAD

**Discussion by rabbit-roger** [fureweb-com/public-google-sheets-parser#32] [Ideas]
Hi,

Now there is difficulty in distinguishing between the following cases:
1. There is a Google Sheets id, but it is not a public table.
2. The table is public but empty.
3. The table is public and there is only one row in it.

In all these cases the parser returns an empty array.

```
const parser = new PublicGoogleSheetsParser(sheetId);
const rows = await parser.parse();
console.log(rows);
// Output: []
```

My suggestion:
1. Return an error.
2. Return an empty array.
3. Return an array of cells from an existing row (not an object).

Or any other implementation. Thanks.

**Comment by fureweb-com** [comment]
Hi, first of all thanks for your interest and suggestions.

The purpose of this library was simply to request a public Google Sheets and get the results as a JSON array.

The idea was that if it was a private sheet, or if there was some other issue that prevented it from getting the data normally, the client would still be able to use it without any exception handling.

So I don't think I'm going to change the implementation at this point.
I would appreciate your understanding of my thinking.

I apologize for being so late in responding.
Have a great day!
